System.IO.Abstractions.TestingHelpers/MockFileData.cs

```diff
@@ -81,6 +81,24 @@ public MockFileData(byte[] contents)
             Contents = contents ?? throw new ArgumentNullException(nameof(contents));
         }
 
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MockFileData"/> class by copying the given <see cref="MockFileData"/>.
+        /// </summary>
+        /// <param name="template">The template instance.</param>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="template"/> is <see langword="null" />.</exception>
+        public MockFileData(MockFileData template)
+        {
+            if (template == null) throw new ArgumentNullException(nameof(template));
```

Comment:
Minor: for consistency for one line conditional blocks do we prefer this syntax or the following:
```
if (condition)
{
    // Statement to execute
}
```

I see the later being used in MockFileData.cs:
https://github.com/System-IO-Abstractions/System.IO.Abstractions/blob/55a82b00781139b041d4a2b9a59a4ad9956d28a7/System.IO.Abstractions.TestingHelpers/MockFile.cs#L31